tests/embroider-css-modules/config/ember-try.js

```diff
@@ -49,7 +49,6 @@ module.exports = async function () {
       },
       {
         name: 'ember-canary',
-        allowedToFail: true,
```

Comment:
Fixed by updating `@ember/test-helpers` to `v3.2.1`.

- https://github.com/emberjs/ember.js/issues/20576
- https://github.com/emberjs/ember-test-helpers/pull/1445